src/site/controller.ts

```diff
@@ -100,7 +107,16 @@ async function getSiteMembers(_req: Request, res: Response) {
 async function assignMemberToSite(_req: Request, res: Response) {
   ExpressHandler(
     async () => {
-      res.sendStatus(501);
+      const { _id } = res.locals.data;
+      const { memberId } = _req.body;
+
+      if (!memberId) return HTTPError('Missing required arg, memberId.', 400);
```

Comment:
Can we make a middlware check for this too? I think you can just use the one you made put pass UserModel? That way we get the check for the memberId existing in the body but not in the db

Reply:
Sure